app/public/cantusdata/templates/admin/map_folios.html

```diff
@@ -321,11 +340,11 @@ <h6><em>Drag and drop a CSV file anywhere<br>on the page to import it.</em></h6>
             for (var i = 1; i < lines.length; i++) {
```

Comment:
leaving a note - you said you wanted to go through this for improved documentation/commenting